### ISSUE TITLE
Support for module-aliasing React to Preact, e.g. with Next.js

### ISSUE THREAD

**Issue by 4cm4k1** [preactjs/preact-devtools#61]
Hi!

Thanks for this wonderful extension.

I am wondering if it is possible to enable some kind of support for module-aliasing React to Preact, e.g. with Next.js, at least until they make Preact support a core Next.js feature as Tim indicated here: https://github.com/zeit/next.js/pull/7890#issuecomment-520267512

For now, I manually alias React to Preact, but it still _looks_ like I’m using React, so the extension doesn’t detect it.

If you need a repo/website to look at, https://github.com/4cm4k1/personal-website and https://anthony.codes have what I describe.

Thanks!!

**Comment by marvinhagemeister** [comment]
This is super awesome! I was missing such a complete test case as your repo. I'll have a look over the following days :+1:

**Comment by 4cm4k1** [comment]
Hey @marvinhagemeister, I have a related question. The docs say to add `import 'preact/debug';` before loading one's app. For Next.js, does that mean in `_app.tsx`? If so, or even elsewhere, it currently causes a compilation error because `window` doesn't exist server-side.

**Comment by marvinhagemeister** [comment]
@4cm4k1 That should be fixed with the next version of Preact. There there is no reference to `window` anymore in `preact/debug`.

Regarding the original issue: I managed to fix all errors I found in the extension with the linked repo a week ago, but the extension is stuck in the review queue in `chrome`. There's unfortunately nothing I can do about that to speed it up and we get no information on how long they need to review it :S

It's very likely that we'll cut a new Preact release this week :+1:
